ports/esp32/mphalport.h

```diff
@@ -136,6 +137,16 @@ static inline int mp_hal_pin_read(mp_hal_pin_obj_t pin) {
 static inline void mp_hal_pin_write(mp_hal_pin_obj_t pin, int v) {
     gpio_set_level(pin, v);
 }
+static inline int mp_hal_pin_output_level(mp_hal_pin_obj_t pin) {
```

Comment:
I suggest calling this `mp_hal_pin_read_output`.

Reply:
Done.